app/views/staffcos/index.html.haml

```diff
@@ -4,9 +4,9 @@
       %h1 Staffing Companies
     #list-item.flex
     - @staffcos.each do |staffcos|
-      #hosp-name
+      #form-name
         %p= staffcos.name
-      #hosp-button
+      #form-button
         %button.button Remove
```

Comment:
do we need button.button? I wonder if just %button will work?

Reply:
the second "button" is the name of its styling